files/ja/web/api/web_authentication_api/index.md

```diff
@@ -35,7 +35,7 @@ create() と get() メソッドが全体像の中でどのように位置づけ
 
 ![ウェブ認証 API のコンポーネントの登録とデータフロー図](webauthn_registration_r4.png)
 
-_図 1 - ウェブ認証による登録手順と各アクションに関連する重要なデータの流れを示す図_
+_図 1 - ウェブ認証による登録手順と各アクションに関連する重要なデータの流れを示す図。_
```

Comment:
Hey @mfuji09. This would cause the ["Emphasis used instead of a heading"](https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md036.md) error with the new version of markdown lint if the whole sentence is emphasized without the tailing punctuation.

As the [en-US document](https://developer.mozilla.org/en-US/docs/Web/API/Web_Authentication_API) has removed the sentence. Is it okay if I add the ignore expression (`<!-- markdownlint-disable -->`) to this sentence for now (in a future update to this article, we can remove the ignore expression)

![image](https://github.com/mdn/translated-content/assets/15844309/2e0da3cc-56d9-4a7d-affa-b97af64278ac)

Reply:
Fixed in 48e4765f6091ffa700d1f67a76d1e45cda0729f4

Reply:
Thank you!